### PR TITLE
Updated URL's to V4

### PR DESCRIPTION
**Pull request by harishrconcur**

### CODE REVIEW

**Review by sp13-hub — APPROVED** [review]
+1

**Review by sp13-hub — APPROVED** [review]
+1

File: src/api-reference/invoice/purchase-request.markdown [hunk 1]
```diff
@@ -41,7 +40,7 @@ https://emea.api.concursolutions.com/purchaserequest/v1/
 
 Once you have registered your application, read about the [Purchase Request API endpoints](/api-reference/invoice/purchase-request-endpoints.html), or click the button to download a request collection for Postman.
```
Comment (charlieconcur) [inline src/api-reference/invoice/purchase-request.markdown]: There should be no links to Postman, cUrl or any other tooling.

**Review by charlieconcur — CHANGES_REQUESTED** [review]
Must remove links to specific tooling.

File: src/api-reference/invoice/purchase-request.markdown (hunk 1 above)
Comment (harishrconcur) [reply src/api-reference/invoice/purchase-request.markdown]: @charlieconcur 
Per your earlier comment on my PR#1477 I added links to post man, do you recommend to remove them now? I see receipt V4 API that has post man links. I do see postman files being valuable for our customers to quickly connect to our API's for verification purpose. 
Please let me know.

Comment (charlieconcur) [reply src/api-reference/invoice/purchase-request.markdown]: Please remove them. I'm working with the Receipts documentation now to remove them there as well.

Comment (harishrconcur) [reply src/api-reference/invoice/purchase-request.markdown]: I have removed references to Postman in 7df33a5. Please check

Comment (harishrconcur) [reply src/api-reference/invoice/purchase-request.markdown]: @charlieconcur Please let us know if this looks good.

Comment (harishrconcur) [reply src/api-reference/invoice/purchase-request.markdown]: @charlieconcur This is good to move to prod, please review and move forward with this.

**Comment by charlieconcur** [comment]
@harishrconcur We need to remove cUrl examples as well -- see the [documentation template](https://raw.githubusercontent.com/concur/developer.concur.com/preview/src/document-template.markdown) for the example.

**Comment by harishrconcur** [comment]
@charlieconcur  I have removed references to cUrl per your inputs in commit 7b155465

File: src/api-reference/invoice/purchase-request-endpoints.markdown [hunk 2]
```diff
@@ -37,66 +39,68 @@ Create a Purchase Request based on provided header and line item details. If the
...
-  "notesToVendor" : "Phase 1 request office chair",
-  "comments" : "Phase 1 request for new employees for office chair",
-  "custom3" : "DEPT",
-  "custom4" : "SALES"
-}
-]
-}
-```
 
+  ```shell
```
Comment (charlieconcur) [inline src/api-reference/invoice/purchase-request-endpoints.markdown]: This should be `json`.

**Comment by charlieconcur** [comment]
Thanks @harishrconcur for resolving that issue. One more fix and then I think we are ready to go.

File: src/api-reference/invoice/purchase-request-endpoints.markdown (hunk 2 above)
Comment (harishrconcur) [reply src/api-reference/invoice/purchase-request-endpoints.markdown]: Changed to json in commit 29fd74fb

**Review by sp13-hub — APPROVED** [review]
+1

**Comment by harishrconcur** [comment]
@charlieconcur I have fixed all of your comments. Thanks

**Review by charlieconcur — APPROVED** [review]

**Pull request merged**